### PR TITLE
pkg/cmdutil: introduce cmdutil

### PR DESCRIPTION
**Pull request by mfojtik**
Used by `oc` and `openshift-apiserver`. Clean extraction from `pkg/cmd/util`.

/cc @deads2k 
/cc @soltysh

### CODE REVIEW

File: pkg/cmdutil/cmd.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	if absName, err := filepath.Abs(filename); err == nil {
+		return absName
+	}
+
+	return filename
+}
+
+// ResolveResource returns the resource type and name of the resourceString.
+// If the resource string has no specified type, defaultResource will be returned.
+func ResolveResource(defaultResource schema.GroupResource, resourceString string, mapper meta.RESTMapper) (schema.GroupResource, string, error) {
```
Comment (deads2k) [inline pkg/cmdutil/cmd.go]: this method is only used by `oc`

File: pkg/cmdutil/cmd.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
+package util
```
Comment (deads2k) [inline pkg/cmdutil/cmd.go]: looks like this whole file is only used in oc

File: pkg/cmdutil/crypto.go [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+	"bytes"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"io/ioutil"
+
+	"k8s.io/client-go/util/cert"
+)
+
+func CertPoolFromFile(filename string) (*x509.CertPool, error) {
```
Comment (deads2k) [inline pkg/cmdutil/crypto.go]: this is only used by an integration test

File: pkg/cmdutil/cmd.go (hunk 2 above)
Comment (soltysh) [reply pkg/cmdutil/cmd.go]: There are a lot of stuff here, and majority is somewhat oc related, I'd not move it, just yet.

**Pull request closed without merging**